recipes/MADL-CPU-IntelMPI/README.md

```diff
@@ -0,0 +1,59 @@
+# MADL-CPU-OpenMPI
+This recipe shows how to run Microsoft Azure Distributed Linear Learner (MADL) on CPUs across
```

Comment:
Add link to the learner landing page when ready.

Reply:
ok